card/mod/Modfile

```diff
@@ -23,10 +23,5 @@ mod 'navbar'
 mod 'google_analytics'
 gem_mod 'edit'
```

Comment:
We need the Modfile to put this before bootstrap? Seems like a small issue we should fix when code gets moved around and into more gems. then this ordering is in the default pack gem.

Reply:
There is no need to worry about ordering in the pack.

`card-mod-bootstrap` will depend on `card-mod-edit`. Then worry not.